src/explorer/utils/validations.ts

```diff
@@ -24,7 +24,7 @@ export function inputValidation(value: string, key: string): string {
     "pricingPolicyId",
     "country",
   ];
-  const specialChars = /[ `!@#$%^&*()+\-=[\]{};':"\\|,.<>/?~]/;
+  const specialChars = /[`!@#$%^&*()+\-=[\]{};':"\\|,.<>/?~]/;
```

Comment:
this may introduce a regression everywhere (e.g all the text fields will allow spaces now) else where this specialChars is used, are we sure we should introduce it here?

Reply:
@xmonader Should I separate the country validation from the rest text fields?

Reply:
I'm not sure, I'm pointing the side effect xD if it could be separated, it would be better imo

Reply:
I added a separate regex for country fields that accepts only alphas & spaces.